### PR TITLE
[Minor Fix] Fix comments in benchmark_serving

### PR DESCRIPTION
**Pull request by gty111**
In `benchmark_serving.py`, `--model` is an required argument, while `--tokenizer` isn't a necessary argument.

When executing the following command
```
python benchmarks/benchmark_serving.py \
        --backend <backend> \
        --tokenizer <your_model> --dataset <target_dataset> \
        --request-rate <request_rate>
```
It will trigger error `benchmark_serving.py: error: the following arguments are required: --model`

### CODE REVIEW

**Review by ywang96 — APPROVED** [review]
Thank you for the quick fix!

Note - I will make the comments easier to read and include more information in https://github.com/vllm-project/vllm/pull/3194

File: benchmarks/benchmark_serving.py [hunk 1]
```diff
@@ -171,9 +171,9 @@ async def benchmark(
     else:
         raise ValueError(f"Unknown backend: {backend}")
 
-    pbar = None if disable_tqdm else tqdm(total=len(input_requests))
-
     print(f"Traffic request rate: {request_rate}")
+
```
Comment (ywang96) [inline benchmarks/benchmark_serving.py:175]: Please remove the whitespace in this line to fix the `yapf` format error. Thanks!

**Review by zhuohan123 — APPROVED** [review]
LGTM! Can you fix the yapf error?

File: benchmarks/benchmark_serving.py (hunk 1 above)
Comment (gty111) [reply benchmarks/benchmark_serving.py]: The whitespace is removed now

**Comment by gty111** [comment]
> LGTM! Can you fix the yapf error?

Sure. The yapf error is fixed now.

**Pull request merged**